actionpack/lib/abstract_controller/translation.rb

```diff
@@ -18,7 +18,9 @@ def translate(key, options = {})
         options[:default] = defaults.flatten
         key = "#{path}.#{action_name}#{key}"
       end
-      I18n.translate(key, options)
+
+      i18n_raise = options.fetch(:raise, Rails.configuration.i18n.raise_on_missing_translations)
```

Comment:
Action Pack shouldn't know about the `Rails` constant since that couples it to the Railties gem, which isn't one of its dependencies - it should be possible to use Action Pack outside of a Rails application.

The way configuration works (as I understand it) is:

 - During initialization, the application makes changes to `Rails.configuration`
 - After initialization, the frameworks' railties copy those values into the frameworks somewhere
 - At runtime, the frameworks use their own copy of the values to make decisions

With that in mind, I think the right thing to do here is to add a `raise_on_missing_translations` setting to `AbstractController::Base`, similar to the one on `ActionView::Base`, and set both from the i18n railtie.